hc/api/tests/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@Sharonsyra after adding the file to `.gitignore` you need to run `git rm -r --cached .` to remove it, since it had already been committed.